plugins/csharp/CLI/Bridge.cpp

```diff
@@ -0,0 +1,33 @@
+#include <msclr/marshal.h>
+#include <msclr/marshal_windows.h>
+
+#include "Bridge.h"
+
+#using MANAGED_PLUGIN_DLL
+
```

Comment:
Apparently, this is used only during compilation. At runtime, the managed DLL (as in the one written in C#) just needs to be reachable in the search path, whatever it is, or simply next to the executable.

Reply:
Should we change the extension of the C++/CLI files?

Reply:
FWICA, it's usually `.cpp` because it's still "C++", just Microsoft-flavoured/extended to not compile to native machine code.

Reply:
That's why I'm asking. I'd almost go for something like `cppcli`or `cccli` or `ccc`. :laughing: